crates/types/src/net/replicated_loglet.rs

```diff
@@ -110,22 +113,33 @@ impl CommonResponseHeader {
 }
 
 // ** APPEND
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize, bilrost::Message)]
 pub struct Append {
     pub header: CommonRequestHeader,
-    pub payloads: Arc<[Record]>,
+    pub payloads: AppendPayloads,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, BilrostAs, Default)]
+#[bilrost_as(dto::AppendPayloads)]
+pub struct AppendPayloads(pub Arc<[Record]>);
```

Comment:
Wondering if we can simply reuse this 

https://github.com/restatedev/restate/blob/a7884cbe045aaee48877aee5eb7b33dd46a18f82/crates/types/src/net/log_server.rs#L299